server/zanata-frontend/src/frontend/package.json

```diff
@@ -5,18 +5,18 @@
   "main": "index.js",
   "scripts": {
     "js": "webpack",
-    "build": "webpack --config webpack.config.js --ignore-scripts=false && npm run build:icons --ignore-scripts=false && npm run build:webpack && npm run build:webpackLegacy --ignore-scripts=false",
```

Comment:
@davidmason , I had to revert the 'ignore-scripts=false' additions to package.json as they stopped the build from working.